.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
+    "prefer-arrow-callback" : "off",
+    "no-underscore-dangle" : "off",
+    "no-unused-vars" : "off",
```

Comment:
In the express, when passing next as parameter but not use that. that why I disabled this.